engine/class_modules/sc_mage.cpp

```diff
@@ -2620,6 +2596,25 @@ struct arcane_mage_spell_t : public mage_spell_t
     return 1.0 + p()->buffs.arcane_charge->check() * ( base + mastery );
   }
 
+  void execute() override
+  {
+    mage_spell_t::execute();
+
+    if ( p()->talents.intuition.ok() && !p()->buffs.intuition->check() && triggers.intuition_blp )
+    {
+      constexpr int blp_threshold = 11;
+
+      p()->state.intuition_blp_count += 1;
+      if ( p()->state.intuition_blp_count >= blp_threshold
+        || ( !background && rng().roll( p()->bugs ? 0.01 : p()->talents.intuition->effectN( 1 ).percent() ) ) )
```

Comment:
I take issue with how `bugs` is used here. The tooltip states it's 5%, with BLP alone it's 9% (and more if you include the base proc chance). Reducing base proc chance to account for extra procs from BLP is fairly normal. The bug here is, arguably, that even with the reduced proc chance the overall proc rate is FAR above the stated 5%.

So either we treat the entire BLP system as a bug or none of it. Picking out just the base chance doesn't make sense here.